File: 프로그래머스/lv0/120822. 문자열 뒤집기/문자열 뒤집기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string) {
    
    for (int i = 0; i < static_cast<int>(my_string.size() / 2); i++)
    {
        char& swap = my_string[my_string.size() - 1 - i];
        char temp = my_string[i];
        my_string[i] = swap;
        swap = temp;

    }
    return my_string;
}

File: 프로그래머스/unrated/181905. 문자열 뒤집기/문자열 뒤집기.cs
using System;
using System.Linq;

public class Solution {
    public string solution(string my_string, int s, int e) {
        var answer = my_string.ToArray();

        Array.Reverse(answer, s, e - s + 1);
        
        return string.Concat(answer);
    }
}